File: 1-Iniciante/1101-Sequencia_de_Numeros_e_Soma-4/1101.py
while True:    
    nuns=input().split(' ')
    nun1=int(nuns[0])
    nun2=int(nuns[1])
    if nun1 <= 0 or nun2 <=0:
        break
    lista=0
    list=[]
    if nun1 < nun2:
        for i in range(nun1, nun2+1):
            lista+=i
            list.append(i)

    else:
        for i in range(nun1, nun2-1, -1):
            lista+=i
            list.append(i)
    
        list.reverse()
    for i in list: print(i, end=" ")
    print(f"Sum={lista}")

File: 1-Iniciante/1101-Sequencia_de_Numeros_e_Soma-4/1101.cs
using System;

public class HelloWorld
{
    public static void Main(string[] args)
    {
        while(true){
            string[] nuns = Console.ReadLine().Split(' ');
            int nun1 = int.Parse(nuns[0]);
            int nun2 = int.Parse(nuns[1]);
            if(nun1 <= 0 | nun2 <= 0){
                break;
            }
            int total = 0;
            if(nun1>nun2){
                for(int i=nun2; i<nun1+1; i++){
                    total+=i;
                    Console.Write($"{i} ");
                }
            }
            else{
                for(int u=nun1; u<nun2+1; u++){
                    total+=u;
                    Console.Write($"{u} ");
                }
            }
            Console.WriteLine($"Sum={total}");
    
        }
    }
}